meson.build

```diff
@@ -30,6 +30,7 @@ sources = [
     'src/Rendering.cpp',
     'src/SpriteRendering.cpp',
     'src/Windowing.cpp',
+    'src/PostProcessRendering.cpp',
```

Comment:
Everything was alphabetically sorted (except Main at the end), try to notice and follow conventions like that.